src/Generate.svelte

```diff
@@ -319,8 +320,8 @@
 
       <label class="flex flex-col">
         <span class="font-bold">Samples</span>
-
-        <input type="number" bind:value={samples} min="1" max={maxSamples} />
+        <!-- most of us have OOM issues with this and this functionality is superceded by the Runs -->
+        <input type="number" disabled class="opacity-30" bind:value={samples} min="1" max={maxSamples} />
       </label>
```

Comment:
this is just me but we should discourage people using the Samples feature since we now have the Copies feature - samples always OOM's for me

Reply:
Agree, I'm thinking to remove it entirely.